modules/extensions/scripts/calico_install.sh

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env bash
```

Comment:
Please ensure this is shellchecked

Reply:
Yes - it's clean. Annotations are included in `modules/extensions/scripts/calico_env.sh` as well, to silence false positives from templated variables.